### ISSUE TITLE
Sub query array arguments not working

### ISSUE THREAD

**Issue by albert-the-creator** [neo4j-graphql/neo4j-graphql-js#132]
Hi, I'm unable to use array type arguments in sub query.

`
  viewer {
    firstName
    itemsByStatus(criteriaF:[OLD, FOO], criteriaB:[BAR, BAZ]){
      name
    }
  }
`

The above will not work. It gets the error 
`Neo4jError: Failed to invoke function `apoc.cypher.runFirstColumn`: Caused by: org.neo4j.cypher.ParameterNotFoundException: Expected parameter(s): criteriaF, criteriaB`

However, when I remove the array input and make them a single arguments, it works. 
`itemsByStatus(criteriaF:OLD, criteriaB:BAR)`

Love this project! Keep up the great work.

**Comment by appinteractive** [comment]
I have the same issue, the result is, that I cant paginate sub objects at all. I would like to use this brilliant library for production, this issue (besides propper ordering) keeps me from doing so.

Please please please give us a working implementation and we will be thankfull like hell 😘

**Comment by kipz** [comment]
This is blocking our move from the neo4j plugin to apollo, so we're keen to get a fix to this too.

**Comment by kipz** [comment]
A bit more data on this one. Arrays as top-level query arguments generate valid cypher, but which can't match. e.g.
```graphql
query team {
  Team(ids: ["someid"]) {
    id
  }
}
```

generates:

```cypher
MATCH (`team`:`Team` {ids:$ids}) RETURN `team` {id: ID(`team`)} AS `team` SKIP $offset
{ offset: 0, first: -1, ids: [ 'someid' ] }
```
Which although doesn't throw an error for us, it does not match. My understanding is that we should be using the `in` syntax here.

**Comment by johnymontana** [comment]
Fixed in #163
